tests/FSharp.Compiler.ComponentTests/Signatures/TestCasesForGenerationRoundTrip/access-minimal-repro.fsx

```diff
@@ -0,0 +1,8 @@
+module Core_access
```

Comment:
Maybe also put into the test module?

Reply:
(same in some other files)

Reply:
There is no standard, the test files simply carried whichever module they had before (minimizing number of changes) and this is a subset of the big access.fsx to demonstrate an issue with sig file gen.